kfdef/kfctl_openshift.yaml

```diff
@@ -72,9 +66,59 @@ spec:
         name: manifests
         path: radanalyticsio/spark/operator
     name: radanalyticsio-spark-operator
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
+        value: opendatahub
+      repoRef:
+        name: manifests
+        path: prometheus/cluster
+    name: prometheus-cluster
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
+        value: opendatahub
+      repoRef:
+        name: manifests
+        path: prometheus/operator
+    name: prometheus-operator
+  - kustomizeConfig:
+      parameters:
+        - name: s3_endpoint_url
+          value: "s3.odh.com"
+      repoRef:
+        name: manifests
+        path: jupyterhub/jupyterhub
+    name: jupyterhub
+  - kustomizeConfig:
+      overlays:
+      #- cuda
+      - additional
+      repoRef:
+        name: manifests
+        path: jupyterhub/notebook-images
+    name: notebook-images
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
+        value: opendatahub
+      repoRef:
+        name: manifests
+        path: airflow/operator
+    name: airflow-operator
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: airflow/cluster
+    name: airflow-cluster
+  - kustomizeConfig:
```

Comment:
I think it would be good to have the example-celery in README, but not in kfdef by default - what is the thought of others - @crobby @nakfour @LaVLaS ? (similarly to Spark, we do not deploy default cluster, I think we should not deploy default Airflow cluster)

Reply:
I'm unfamiliar enough with Airflow that I can't say whether or not it makes sense to have a "default" cluster.  In the Kafka case, we DO spin-up a kafka instance, but in Spark, we do not.  I guess here is my thought process...I don't know the right answer:  Is the "default" cluster something that would generally be usable by a typical user, or would a typical user wind-up needing to customize a lot of things?  If we'd expect most users to wind-up generating a cluster that is == our what our default cluster would be, then maybe just creating the cluster right away is the better option.

Reply:
Ah, sorry, should have provided more background. The "default" cluster is really only useful to take a look what gets spin up and how the UI looks like. you need to at least customize the DAGs repository (Vedant will add a parameter for it later), but I believe users will always need to customize the repo URL and in mos cases also number of replicas and potentially other bits, so I believe it is not worth to deploy the default cluster - that's also why we called it "example-celery" since it is really just example. I also asked Vedant to add `example-kube-executor` in the future. IOW I vote for no default cluster

Reply:
Ok, that makes sense to me then.  I am also a "no" for default cluster.